brokerapi/brokers/cloudsql/broker.go

```diff
@@ -524,23 +522,17 @@ func (b *CloudSQLBroker) Unbind(creds models.ServiceBindingCredentials) error {
 
 // PollInstance gets the last operation for this instance and checks its status.
 func (b *CloudSQLBroker) PollInstance(instanceId string) (bool, error) {
-
-	instance, err := db_service.GetServiceInstanceDetailsById(instanceId)
-	if err != nil {
-		return false, brokerapi.ErrInstanceDoesNotExist
-	}
-
```

Comment:
This is already checked by the code that calls this method, so no need to validate again.